sound/soc/sof/intel/Kconfig

```diff
@@ -99,15 +99,15 @@ endif ## SND_SOC_SOF_INTEL_ACPI
 
 if SND_SOC_SOF_INTEL_PCI
 
-config SND_SOC_SOF_EDISON_SUPPORT
-	bool "SOF support for Tangier/Edison"
+config SND_SOC_SOF_MERRIFIELD_SUPPORT
+	bool "SOF support for Tangier/Merrifield"
 	help
           This adds support for Sound Open Firmware for Intel(R) platforms
           using the Baytrail, Braswell or Cherrytrail processors.
```

Comment:
Hmm... How BYT and CHT are related to Merrifield here?

Reply:
And BSW.

Reply:
copy/paste, gah. thanks for spotting this one, there is a point where your eyes glaze over this sort of mistakes.